code/processes/dqe.q

```diff
@@ -56,6 +56,14 @@ nullchk:{[t;colslist;thres]
   update thresholdfail:nullspercentage>thres from res                                                           /- compare each column's nulls percentage with threshold thres
   }
 
+rangechk:{t;colslist;tlower;tupper;thres]                                                                       /- check that values are within the range defined by tlower and tupper tables
+  colslist:((),colslist) except exec c from meta t where t in "csSC ";                                          /- exclude columns that do not have pre-defined limits
+  tupper:((),colslist)#tupper;
+  tlower:((),colslist)#tlower;
+  d:(sum (tt within (tlower;tupper))*100%count tt:((),colslist)#t;                                              /- dictionary with results by columns
```

Comment:
Don't need the `((),colslist)` in these 3 lines, can just use `colslist` as it's already guaranteed to be a list from first line in function

Reply:
Thanks! Fixed